### PR TITLE
[WFLY-18856] Document stability levels

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.redhat.com/browse/WFLY-18856

### CODE REVIEW

**Review by pferraro — CHANGES_REQUESTED** [review]
Minor edits

File: docs/src/main/asciidoc/_admin-guide/Feature_stability_levels.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+The WildFly project has high standards related to quality, stability and backwards compatibility. A key way an open source project like WildFly can ensure high standards are met is by "community bake" -- allowing interested users to have access to features that are still undergoing a hardening process, while not forcing users who are not interested in such things to consume them.
+
+To better facilitate this, WildFly 31 introduced the notion of formal "stability levels" that can be associated with functionality. When starting a WildFly process, users can use the `--stability` command line param to control the minimum level of available features, with a value of `experimental`, `preview`, `community` or `default`.
```
[the remark this answers is not in the document]
Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Feature_stability_levels.adoc]: Fixed.

File: docs/src/main/asciidoc/_admin-guide/Feature_stability_levels.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+----
+
+Features at a lower stability level will not be available for use.
+
+A WildFly installation will have a standard stability level, determined by the xref:Galleon_Guide.adoc#WildFly_Galleon_feature-packs[Galleon feature-pack] used to provision the installation. This level is used if the `--stability` param is not set. For a standard WildFly installation, this level is `community`. For WildFly Preview it is `preview`.
+
+Some details on the stability levels:
+
+* *experimental* -- This level is for true bleeding edge functionality that may never advance to a higher stability level. No WildFly feature-pack or distribution zip/tar would enable this level by default.
+* *preview* -- This is the level for features at a sufficient level to be available by default in WildFly Preview, but not in standard WildFly. The general expectation for features at this level is that they will eventually move to `community` level in substantially similar form (although this is not guarranteed).
```
[the remark this answers is not in the document]
Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Feature_stability_levels.adoc]: Fixed.

File: docs/src/main/asciidoc/_admin-guide/Feature_stability_levels.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* *community*-- This is the level for features at a sufficient level to be available by default in standard WildFly. Features at this level are not expected to change incompatibly over time in a manner inconsistent with the expectations of the xref:Galleon_Guide.adoc#WildFly_Galleon_feature-packs[Galleon feature-pack] that provides them.
```
[the remark this answers is not in the document]
Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Feature_stability_levels.adoc]: Fixed.

**Review by pferraro — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @pferraro

**Pull request merged**